.github/workflows/ci.yaml

```diff
@@ -14,7 +14,10 @@ jobs:
         with:
           go-version: '1.15'
       - uses: actions/checkout@v1
-      - run: go mod download
+      - uses: actions/cache@v2
+        with:
+          key: 2-${{ runner.os }}-go-${{ hashFiles('go.sum') }}
+          path: ~/go/pkg/mod
```

Comment:
How does this action replace the functionality of 'go mod download' ?

Reply:
It doesn't. I'd just removed the `go mod download` step since it isn't particularly useful for us.